src/app.types.tsx

```diff
@@ -32,32 +32,39 @@ export interface CatalogueCategory {
   is_leaf: boolean;
   catalogue_item_properties?: CatalogueCategoryFormData[];
 }
+export interface AddManufacturer {
+  name: string;
+  url?: string;
+  address: Address | undefined;
+  telephone?: string;
+}
 
-export interface Manufacturer {
+export interface AddManufacturerResponse {
```

Comment:
```suggestion
export interface Manufacturer {
```
Shouldn't this replace the Manufacturer below? It is what is returned when getting or doing the post.

Reply:
the interface does not have code though, a post request returns a manufacturer that includes a code. Maybe that was my mistake but that's why they are separate interfaces

Reply:
The responses of the post, patch or get request is all the same object, they each have code regardless just the one below doesn't (which wont error even though it is being returned by the api), so it can be replaced with one that does include the code.